action.yml

```diff
@@ -20,13 +20,15 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Install NPM deps
-      run: npm install
-      shell: bash
+    - uses: actions/checkout@v2
```

Comment:
We're now more explicit about the setup for this and pinning node to version 16.